libvirt.go

```diff
@@ -918,6 +934,72 @@ func (l *Libvirt) Version() (string, error) {
 	return versionString, nil
 }
 
+// Reboot reboots the domain
```

Comment:
this comment can be improved, it should also be a full sentence. Add a note about the behavior when flags is 0.